components/dashboard/src/settings/Billing.tsx

```diff
@@ -10,8 +10,6 @@ import { BillingAccountSelector } from "../components/BillingAccountSelector";
 export default function Billing() {
     return (
         <PageWithSettingsSubMenu title="Billing" subtitle="Usage-Based Billing.">
-            <h3>Usage-Based Billing</h3>
-            <h2 className="text-gray-500">Manage usage-based billing, spending limit, and payment method.</h2>
             <div className="mt-8">
```

Comment:
**nitpick**: Irrelevant to the changes here, but let's remove the top margin here as the first section does not need it.

```suggestion
            <div>
```